stdlib/private/StdlibUnittest/StdlibUnittest.swift

```diff
@@ -112,29 +128,27 @@ func _printStackTrace(_ stackTrace: SourceLocStack?) {
   }
 }
 
-// FIXME: these variables should be atomic, since multiple threads can call
-// `expect*()` functions.
-var _anyExpectFailed = false
-var _seenExpectCrash = false
+var _anyExpectFailed = AtomicBool(false)
+var _seenExpectCrash = AtomicBool(false)
 
 /// Run `body` and expect a failure to happen.
 ///
-/// The check passes iff `body` triggers one or more failures.
```

Comment:
This `iff` wasn't a typo -- it's [a shorthand for "if and only if"](https://en.wikipedia.org/wiki/If_and_only_if).

Reply:
Oops, sorry 😅